### PR TITLE
KFSPTS-6313: reskin IWant document

### PR DESCRIPTION
**Pull request by bhutchinson**

### CODE REVIEW

**Comment by bhutchinson** [comment]
@db483 These changes are to apply the new KFS 7.0 look & feel to the I Want document on our upgrade branch. I can show you what that looks like in my local dev env if you'd like. Please let me know if you have any questions.

File: src/main/java/edu/cornell/kfs/module/purap/document/web/struts/IWantDocumentForm.java [hunk 1]
```diff
@@ -156,28 +154,6 @@ public String getNewAdHocRoutePersonIdForLookup() {
...
-            iwntTitle = "I Want Document Step #4";
-        } else if (CUPurapConstants.IWantDocumentSteps.REGULAR.equals(step)) {
-            //iwntTitle = getDocument().getDocumentTitle();
-            iwntTitle = KRADServiceLocatorWeb.getDataDictionaryService().getDocumentLabelByClass(IWantDocument.class);
-        }
-        return iwntTitle;
-    }
-
-    /**
      * This overridden method ...
```
Comment (db483) [inline src/main/java/edu/cornell/kfs/module/purap/document/web/struts/IWantDocumentForm.java]: could probably also remove this method comment that is not very useful

File: src/main/webapp/jsp/module/purap/iWantDocumentFinish.jsp [hunk 2]
```diff
@@ -1,31 +1,13 @@
...
-	</div>
+<%@ include file="/jsp/sys/kfsTldHeader.jsp"%>
+
+<kul:page showDocumentInfo="false"
+          headerTitle="I Want Document"
+          docTitle="I Want Document Submitted"
+          transactionalDocument="false"
+          htmlFormAction="purapIWant"
+          errorKey="*">
+    <br/>
```
Comment (db483) [inline src/main/webapp/jsp/module/purap/iWantDocumentFinish.jsp]: this <br/> tag does not seem aligned correctly but I don't know if it's worth changing it

**Comment by db483** [comment]
Code changes look good and tests passed, merging.

**Pull request merged**